### PR TITLE
OCPBUGS#6709: Misleading information in Driver Toolkit doc in 4.11

### PR DESCRIPTION
**Pull request by tshwartz**
**Title:**
Misleading information in Driver Toolkit doc in 4.11

Summary of changes:
Updates to step 1 of the 'Finding the Driver Toolkit image URL in the payload' section.

Versions:
Enterprise 4.11

Issue:
(https://issues.redhat.com/browse/OCPBUGS-6709?filter=-1)

Link to docs preview:
http://file.emea.redhat.com/tshwartz/OCPBUGS-6709/hardware_enablement/psap-driver-toolkit.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Fixmetal** [comment]
/lgtm
thanks @tshwartz

File: modules/psap-driver-toolkit-pulling.adoc [hunk 1]
```diff
@@ -28,17 +28,29 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
...
+# For x86 image:
+$ oc adm release info quay.io/openshift-release-dev/ocp-release:{product-version}.z-x86_64 --image-for=driver-toolkit
+
+# For ARM image:
+$ oc adm release info quay.io/openshift-release-dev/ocp-release:{product-version}.z-aarch64 --image-for=driver-toolkit
 ----
 +
+[IMPORTANT]
+====
+ARM images are built for 64-bit Arm (AArch64) only, and are currently a Technology Preview feature. For more information about {product-title} on ARM, see xref:../release_notes/ocp-4-11-release-notes.html#ocp-4-11-release-notes[Release Notes]. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
```
Comment (kcarmichael08) [inline modules/psap-driver-toolkit-pulling.adoc:40]:
```suggestion
ARM images are built for 64-bit ARM (AArch64) only, and are currently a Technology Preview feature. For more information about {product-title} on ARM, see xref:../release_notes/ocp-4-11-release-notes.html#ocp-4-11-release-notes[Release Notes]. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. 
```

Comment (kcarmichael08) [reply modules/psap-driver-toolkit-pulling.adoc:40]: Maybe there is an exception for Telco docs (?) but usually xrefs aren't allowed in OCP modules (Release Notes link).

File: modules/psap-driver-toolkit-pulling.adoc [hunk 2]
```diff
@@ -28,17 +28,29 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 +
 [source,terminal]
 ----
-$ oc adm release info 4.11.0 --image-for=driver-toolkit
+# For x86 image:
```
Comment (kcarmichael08) [inline modules/psap-driver-toolkit-pulling.adoc:31]: I think you need an explicit instruction to "run" or "enter" the command, and only one command per code block is allowed per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Maybe something like:

The image URL of the `driver-toolkit` corresponding to a certain release can be extracted from the release image using the `oc adm` command. 

For an x86 image, enter the following command:

`[example]`

For an ARM image, enter the following command:

`[example]`

**Review by kcarmichael08 — COMMENTED** [review]
I had a couple of comments/questions; otherwise, LGTM!

File: modules/psap-driver-toolkit-pulling.adoc (hunk 1 above)
Comment (tshwartz) [reply modules/psap-driver-toolkit-pulling.adoc:40]: Hi @kcarmichael08 Thanks for your feedback. I see other links in the section so I'll ping you later to finalize this.

**Comment by tshwartz** [comment]
Please wait before merging - checking one last thing.

**Pull request merged**